src/routes/(app)/person/[id]/+page.svelte

```diff
@@ -19,6 +19,9 @@
   let personId: number | undefined;
   let person: TMDBPersonDetails | undefined;
   let pageError: Error | undefined;
+  let sortOption = "Vote count";
+  let cachedCredits: TMDBPersonCombinedCredits | undefined;
+  let sortedCredits: TMDBPersonCombinedCredits | undefined;
```

Comment:
Instead of having two lists, we could just have one `credits` array and sort it in place whenever a the sort is changed?